sql/core/src/main/scala/org/apache/spark/sql/execution/command/functions.scala

```diff
@@ -74,6 +76,15 @@ case class CreateFunctionCommand(
   }
 
   override def run(sparkSession: SparkSession): Seq[Row] = {
+    // Checks if the given resources exist
+    val hadoopConf = sparkSession.sparkContext.hadoopConfiguration
+    val nonExistentResources = resources.filter { r =>
+      val path = new Path(r.uri)
+      !path.getFileSystem(hadoopConf).exists(path)
```

Comment:
I am not sure whether this could cause a false alarm. For example, the security access exceptions. 

You can check the impl of addJar or addFile.

Reply:
cc @zsxwing

Reply:
I think we should better follow the logic in `addJar`
https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/SparkContext.scala#L1843